src/app/components/dashboard-apps-dock/dashboard-apps-dock.coffee

```diff
@@ -0,0 +1,157 @@
+
+DashboardAppsDockCtrl = ($scope, $cookies, $uibModal, $window, MnoeOrganizations, MnoeAppInstances) ->
+  'ngInject'
+
+  $scope.appDock = {}
+  $scope.appDock.isMinimized = true
+  $scope.activeApp = null
+  $scope.launchApp = {isClosed: true}
+
+  # 'Lock' the dock when a menu is expanded.
+  # Ie: we disable all effects and animation
+  $scope.appDock.isAnimated = () ->
+    $scope.activeApp == null
+
+  # Scope initialization
+  $scope.modal ||= {}
+
+  # Mobile dock: max-height need to be an absolute value for the scrolling to work
+  pageHeight = jQuery(window).height()
+  navHeight = pageHeight * 0.7
+  jQuery('#mobile-dock').css({ maxHeight: navHeight + 'px' })
+
+  # ----------------------------------------------------------
+  # Permissions helper
+  # ----------------------------------------------------------
+  $scope.helper = {}
+  $scope.helper.displaySettings = ->
+    MnoeOrganizations.can.update.appInstance()
+
+  $scope.helper.dataSyncPath = (instance) ->
```

Comment:
I don't think this is used as the button is on the settings modal (which define also this function).